opendatahub-io/odh-dashboard
# Fix gpu fileread

Adds a timeout to the SAToken Promise to prevent dev_mode issues stemming from #573 

## Description


## How Has This Been Tested?
1. Verify branch works (access api/gpu endpoint) locally `npm run dev`
2. Verify branch works on cluster with ODH when built

## Merge criteria:
